rsconf/package_data/rsconf/rsconf.sh

```diff
@@ -572,8 +572,15 @@ rsconf_yum_install() {
         cmd=reinstall
     fi
     if (( ${#todo[@]} > 0 )); then
-        yum "$cmd" --color=never -y -q "${todo[@]}"
+        rsconf_yum "$cmd" "${todo[@]}"
     fi
 }
 
+
+rsconf_yum() {
```

Comment:
I think `install_yum` does what you want.

Reply:
da85b31